forte/version.py

```diff
@@ -19,3 +19,4 @@
 VERSION_SHORT = "{0}.{1}".format(_MAJOR, _MINOR)
 VERSION = "{0}.{1}.{2}".format(_MAJOR, _MINOR, _REVISION)
 FORTE_IR_VERSION = "0.0.1"
+PACK_VERSION = [0, 0, 1]
```

Comment:
We should probably also use the `Major` `Minor` `Revision` style for pack version too.